addons/website_livechat/__manifest__.py

```diff
@@ -42,6 +42,26 @@
             'im_livechat/static/src/legacy/public_livechat.scss',
             'im_livechat/static/src/legacy/public_livechat_chatbot.scss',
             'website_livechat/static/src/legacy/public_livechat.scss',
+
+            'mail/static/src/utils/*.js',
+            'mail/static/src/js/emojis.js',
+            'mail/static/src/component_hooks/*.js',
+            'mail/static/src/model/*.js',
+            'mail/static/src/models/*.js',
+            'im_livechat/static/src/models/*.js',
+            'mail/static/src/services/messaging_service.js',
+            # Framework JS
+            'bus/static/src/js/*.js',
+            'bus/static/src/js/services/bus_service.js',
+            'bus/static/src/js/services/legacy/legacy_bus_service.js',
+            'web/static/lib/luxon/luxon.js',
+            'web/static/src/core/**/*',
+            # FIXME: debug menu currently depends on webclient, once it doesn't we don't need to remove the contents of the debug folder
+            ('remove', 'web/static/src/core/debug/**/*'),
+            'web/static/src/env.js',
+            'web/static/src/legacy/js/core/misc.js',
+            # 'web/static/src/legacy/js/env.js',
+            'im_livechat/static/src/public/main.js',
```

Comment:
Each of these should probably be defined in the `assets_frontend` in their respective module instead?

For our code we will quickly enough need them for portal chatter.

For web it seems generic enough to be always there.

Reply:
changes done